api/src/main/java/org/pmiops/workbench/genomics/WgsCohortExtractionService.java

```diff
@@ -124,6 +133,33 @@ public TerraJob submitGenomicsCohortExtractionJob(DbWorkspace workspace, Long co
                             .put(
                                 "WgsCohortExtract.wgs_extraction_temp_tables_dataset",
                                 "\"" + cohortExtractionConfig.extractionTempTablesDataset + "\"")
+                            .put(
+                                "WgsCohortExtract.wgs_intervals",
+                                "\"gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.interval_list\"")
+                            // This value will need to be dynamically adjusted through testing
+                            .put("WgsCohortExtract.scatter_count", "1000")
+                            .put(
+                                "WgsCohortExtract.reference",
+                                "\"gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta\"")
+                            .put(
+                                "WgsCohortExtract.reference_index",
+                                "\"gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai\"")
+                            .put(
+                                "WgsCohortExtract.reference_dict",
+                                "\"gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dict\"")
+                            // Will produce files named "interval_1.vcf.gz", "interval_32.vcf.gz",
```

Comment:
A typical pattern in batch processing is to include the shard index + shard count. I don't know if the workflow supports this or not currently, e.g. `{basename}-001-of-032.vcf.gz

Also, should these start at 0, or 1?